apps/web/src/lib/services/healthlake-conflict-resolver.ts

```
@@ -0,0 +1,506 @@
+
+      case 'remote_wins':
+        return {
+          field: conflict.field,
+          action: 'accept_remote',
+          value: conflict.remoteValue,
+          reason: 'Remote version prioritized by strategy'
+        };
+
+      case 'timestamp_based':
+        const useRemote = conflict.lastModified.remote > conflict.lastModified.local;
+        return {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)